### ISSUE TITLE
Enderian: Particles

### ISSUE THREAD

**Issue by Faeriin** [apace100/origins-fabric#128]
I've noticed that when playing as an Enderian the end particles don't seem to appear.
![2021-02-23_15 55 31](https://user-images.githubusercontent.com/45500702/108803788-a7eea300-75ef-11eb-855e-60f4b78fa9cf.png)

**Comment by apace100** [comment]
Yes. The particles don't appear for the player who has the origin, only for other players. I'll mark this as enhancement so I'll remember that people want to see the particles on themselves as well. :D

**Comment by Faeriin** [comment]
:D Thank you!
